docs/components/concepts/messages.md

```diff
@@ -49,6 +49,13 @@ zbctl publish message "Money collected" --correlationKey "order-123" --ttl 1h
    </p>
  </details>
 
+## Message expiration
+
+When a buffered message could not be correlated to a process instance in the
+given time-to-live, it gets removed from the buffer afterwards. Internally, the
```

Comment:
❌ This is duplicate because it is already discussed in the `Message buffering` section directly above this.

>A message has a **time-to-live** (**TTL**) which specifies for how long it's buffered. Within this time, the message can be correlated to a process instance.